src/update.c

```diff
@@ -270,7 +270,7 @@ static int main_update()
 				fclose(verfile);
 			}
 
-			if (run_command("/usr/bin/mixin"
+			if (run_command("/usr/bin/mixin "
```

Comment:
Hmm, sorry, I reviewed too quickly. I think the intention was for `"/usr/bin/mixin"` and `"build"` to be separate arguments (because `run_command_full` accepts varargs).

Does the code work with the separate arguments (i.e. `run_command("/usr/bin/mixin", "build", ...)`)

Reply:
Yes, you are right. I will update this.